spring/spring-data/src/test/java/io/quarkus/ts/spring/data/primitivetypes/CommonsHeadersIT.java

```diff
@@ -63,6 +65,8 @@ public void testRequestScope() {
     //This is for regression test for https://github.com/quarkusio/quarkus/pull/12234
     @Test
     @DisplayName("session scope")
+    @DisabledOnQuarkusVersion(version = "2.3.*", reason = "quarkus-spring-data-rest is now reactive")
```

Comment:
Is it temporary fix? Because it looks like https://github.com/quarkusio/quarkus/issues/12233 is about spring DI and should be orthogonal to spring-data

Reply:
As it now uses reactive approach, I would say it's a permanent fix. Wdyt @jsmrcka ?

Reply:
TBH, I don't understand why this doesn't work with reactive approach.

And also, why is it disabled just for `2.3.*`? I suppose it's not going to work for any future version.

Reply:
This is a good point... as I dropped the undertow dependency, this is not going to work ever, so we should delete this test case. 

It does not work because the reactive approach does not propagate the session id.